TrafficCapture/dockerSolution/README.md

```diff
@@ -16,4 +16,33 @@ it will be mounted to the traffic-comparator and jupyter services.
 
 Netcat is still used to connect several of the components and we're still working on improving the resiliency story
 between these containers. The long term approach is to replace fixed streams with message bus approaches directly (i.e.
-Kafka).  In the short term, we can and are beginning, to leverage things like conditions on dependent services.
\ No newline at end of file
+Kafka).  In the short term, we can and are beginning, to leverage things like conditions on dependent services.
+
+### Running the Docker Solution
+
+While in the TrafficCapture directory, run the following command:
+
+`./gradlew :dockerSolution:composeUp`
+
+### Compatibility
+
+Must have Java version 11 installed.
+
+The tools in this directory can only be built if you have Java version 11 installed.
+
+The version is specified in `TrafficCapture/build.gradle` using a Java toolchain, which allows us
+to decouple the Java version used by Gradle itself from Java version used by the tools here.
+
+Any attempt to use a different version will cause the build to fail and will result in the following error (or similar)
+depending on which tool/project is being built. The below example shows the error printed when running e.g `./gradlew
```

Comment:
"The below example shows the error printed when running"

-->
Below is an example error when attempting to build with an incompatible Java version.